### PR TITLE
Update some gems

### PR DESCRIPTION
**Pull request by Frost**
- cancan
- execjs
- haml
- uglifier
- simple-navigation
- net-ldap
- kaminari
- friendly_id
- headless
- pry-nav
- pry-rails
- formtastic
- jquery-rails
- factory_girl
- bootstrap
- _Remove_ `decent_exposure`
- _Remove_ `nokogiri`

### CODE REVIEW

File: spec/models/mage/voucher_spec.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 require 'spec_helper'
 describe Mage::Voucher do
   before(:each) do
-    stub_request(:post, 'http://localhost:8981/solr/update?wt=ruby').to_return(status: 200, body: '')
+    stub_request(:post, 'http://localhost:8981/solr/update?wt=ruby').
```
Comment (Koronen) [inline spec/models/mage/voucher_spec.rb]: I think we can remove this stub, since we've migrated the search functionality from Solr to Postgres.

Comment (Frost) [inline spec/models/mage/voucher_spec.rb]: Agreed. Also, the rest of that file looks horrible :/

Comment (Frost) [inline spec/models/mage/voucher_spec.rb]: It's removed, along with a bunch of similar ones.

**Comment by Frost** [comment]
...and by "some" gems, I actually mean quite a lot of them, including FactoryGirl, Formtastic and Bootstrap, along with whatever view changes were necessary for that to happen.

This also includes @Koronen's branch for getting rid of `less-rails-bootstrap` and replacing it with `bootstrap-sass`.

I kinda want to have this merged before we go ahead further and bump the RSpec and Rails versions, since those two probably require a lot more manual work.

**Comment by Yopi** [comment]
LGTM

**Pull request merged**